src/upload/application/views/buildings/buildings_buildlist_script.php

```diff
@@ -10,20 +10,17 @@ function t() {
 	s           = ss - aa;
 	m           = 0;
 	h           = 0;
+	gurl        = "game.php?page={call_program}&planet=" + pl;
 	if ( (ss + 3) < aa ) {
-		blc.innerHTML = "{bd_finished}<br>" + "<a href=game.php?page={call_program}&planet=" + pl + ">{bd_continue}</a>";
+		blc.innerHTML = "{bd_finished}<br>" + "<a href=" + gurl + ">{bd_continue}</a>";
 		if ((ss + 6) >= aa) {
-			window.setTimeout('document.location.href="game.php?page={call_program}&planet=' + pl + '";', 3500);
+			window.setTimeout('document.location.href="' + gurl + '";', 3500);
 		}
 	} else {
-		if ( s < 0 ) {
-			if (1) {
-				blc.innerHTML = "{bd_finished}<br>" + "<a href=game.php?page={call_program}&planet=" + pl + ">{bd_continue}</a>";
-				window.setTimeout('document.location.href="game.php?page={call_program}&planet=' + pl + '";', 1000);
-			} else {
-				timeout = 0;
-				blc.innerHTML = "{bd_finished}<br>" + "<a href=game.php?page={call_program}&planet=" + pl + ">{bd_continue}</a>";
-			}
+		if ( s <= 0 ) {
```

Comment:
@duhow why are you replacing `s < 0` to `s <= 0`?

Reply:
Just to show as complete once the timer has 1 second left.
Otherwise it would display `00:00:00` -> `Finished`